src/components/FormValidator.js

```diff
@@ -3,13 +3,13 @@ export default class FormValidator {
     this._formElement = formElement;
     this._formSelector = options.formSelector;
     this._inputSelector = options.inputSelector;
-    // this.inputList = Array.from(
-    //   this._formElement.querySelectorAll(this._inputSelector)
-    // );
+    this.inputList = Array.from(
+      this._formElement.querySelectorAll(this._inputSelector));
     this._submitButtonSelector = options.submitButtonSelector;
     this._inactiveButtonClass = options.inactiveButtonClass;
     this._inputErrorClass = options.inputErrorClass;
     this._errorClass = options.errorClass;
+    this._ButtonElement = this._formElement.querySelector(this._submitButtonSelector);
```

Comment:
`_ButtonElement`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)

But you don't use it. Please, delete it